Lombiq.HelpfulLibraries.OrchardCore/Contents/ContentManagerSessionExtensions.cs

```diff
@@ -81,8 +81,9 @@ public static class ContentManagerSessionExtensions
     {
         List<ContentItem> contentItems = null;
         List<ContentItem> storedItems = null;
+        var contentItemIdsArray = contentItemIds.ToImmutableArray();
```

Comment:
This is not good, because neither list nor array implements `ImmutableArray<T>` so this will almost certainly copy.
![image](https://user-images.githubusercontent.com/4007293/212091295-f7c16d24-87e1-4aa5-98cc-e58c9c6b0862.png)

Rather use `contentItemIds.AsList()` here too.